website/docs/r/kv_secret_v2.html.md

```diff
@@ -69,6 +69,11 @@ The following arguments are supported:
 * `data_json` - (Required) JSON-encoded string that will be
   written as the secret data at the given path.
 
+* `custom_metadata` - (Optional) A nested block containing configuration options for the
```

Comment:
Isn't this `map[string]string` ? Not sure what is meant by "nested block".

Reply:
Added docs to clarify the configuration structure in e5eef1f